app/views/records/edit_fields/_creator_organizational.html.erb

```diff
@@ -1,5 +1,15 @@
-<% # This is gross, but because the authorities have no unique ID, but instead an ID and title that are the same and %>
-<% # that change depending on the spelling, we need a way to ensure the partial is included and this is as good as I can think of. %>
-<% # The reason issue is that the ID of the authority needs to stay the same, even if the title changes %>
-<%= render "records/edit_fields/creator_organisational",f: f, type: type, template: template, hash: hash %>
+<div data-toggleable-group="<%= type %>">
+	<% # TODO: workout why setting the name fields as required causes the deposit form to not register the meta data was entered %>
+	<% [
+		{ f: f, field_type: :text, field_slug: :creator_organization_name, field_args: { data: { required: f.object.required?(:creator) } } },
+		{ f: f, field_type: :text, field_slug: :creator_ror },
+		{ f: f, field_type: :text, field_slug: :creator_grid },
+		{ f: f, field_type: :text, field_slug: :creator_wikidata }
+	].each do |field| %>
+		<% options = { field_group: :creator, template: template, hash: hash }.merge(field) %>
```

Comment:
The code is fine, but I am sure you will love `with_options` https://apidock.com/rails/Object/with_options